modules/op-using-custom-scc-to-improve-prod-security.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: PROCEDURE
```

Comment:
missing list of assemblies where this module is included; please see mod docs templates